fili-core/src/main/java/com/yahoo/bard/webservice/web/responseprocessors/EtagCacheResponseProcessor.java

```diff
@@ -113,6 +113,11 @@ public void processResponse(JsonNode json, DruidAggregationQuery<?> druidQuery,
             }
         }
 
+        ((ObjectNode) json).put(
```

Comment:
Wouldn't this type of metadata belong in the header, rather than the body of the response?

Reply:
Yes, this type of metadata is actually in the header at the moment. If I were going to put it in body of the response, I would have done something like 

```
body = json.get(DruidJsonResponseContentKeys.RESPONSE.getName())
body.put("cachedResponse", true/false)
```

Reply:
sorry I got a bit confused. It looks like we are augmenting the Druid response, not the Fili response. This comment no longer applies.